src/debugging/netcore/NetCoreDebugHelper.ts

```diff
@@ -94,11 +97,34 @@ export class NetCoreDebugHelper implements DebugHelper {
                 netCore: {
                     appProject: unresolveWorkspaceFolder(options.appProject, context.folder)
                 }
+            },
+            {
+                name: 'Docker .NET Core Attach',

```

Comment:
Are we sure we want to add this by default? It stuffs more into launch.json; I thought we wanted to have _less_.

Reply:
Yes that is the plan we agreed on when we discussed this feature. Otherwise user has know about this and do one extra step to generate this config.

Reply:
I wasn't here when we discussed this feature 😄 
I offer a vote of dissent. We're placing front and center an experience we know is not great, meant only as an arbitrary solution for those who can't use the "proper" experience.

Reply:
Additionally, aside from automatically installing the debugger for .NET Core, I'm not really sure we're offering anything that they couldn't accomplish themselves with a simple `coreclr` `attach` configuration...

Reply:
This simplifies the user experience of preparing the container for debugging and constructing the coreclr attach with right settings.

Reply:
I'm not sure I understand what we're trying to solve here. IMO the value add is not simplicity--attach is already off the happy path and therefore is the advanced case, consequently doing a Docker attach is not really any simpler than doing an CoreCLR attach. There _is_ a clear value add in terms of setting up and installing the debugger, but unfortunately it takes a lot of time even in the best case--whereas, with proper setup, a CoreCLR attach config could include debugger setup out-of-band so as not to slow down F5.

Honestly, I think we would be better off providing a guide on how to use the CoreCLR attach, and guidance or even scripts that would assist in setting up the debugger. I'm concerned about the added complexity to our debugging experience, and especially concerned about scaffolding this by default.

Reply:
Now it is added only thru add new config.